app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
   has_many :posts, dependent: :destroy
   has_many :comments, dependent: :destroy
```

Comment:
- [ ] Great job adding the `devise gem` and customizing your layouts Team. But I notice your layout needs some arrangement to look like the given wireframe. please make sure all the form elements are aligned to the center and are equal in length as shown on the wireframe. 
### Given wireframe 
![blog_login](https://github.com/MosDevx/BlogApp/assets/69077061/13907165-45d6-4d04-b042-7a8dbf2e97e0)


### Your design 
![image](https://github.com/MosDevx/BlogApp/assets/69077061/ac1275bb-1f8a-4b00-9de5-092bd33af6dd)

- [Optional] Also, once the app starts it crashes when I try to reload unless I follow the provided link, This Is not a good idea according to a user experience, Please take a look.

https://github.com/MosDevx/BlogApp/assets/69077061/83edd347-07eb-4d1b-8331-5ad2f0f15dd0

Reply:
Hey @TadesseDev . Thanks for the feedback. 
Unfortunatel I've been unable to recreate this bug.
Did it happen consistently?

Reply:
Hello @MosDevx, yes it keeps happening even after I switch to a new device. Please check with a different computer than the one you used for development. If it works fine by then, Let the second reviewer review this, please.